File: app/src/main/java/br/com/star_wars_wiki/entity/ResponseFavorite.java
package br.com.star_wars_wiki.entity;

import com.google.gson.annotations.SerializedName;

public class ResponseFavorite {
    String status;
    String message;
    String error;
    @SerializedName("error_message")
    String errorMessage;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getError() {
        return error;
    }

    public void setError(String error) {
        this.error = error;
    }

    public String getErrorMessage() {
        return errorMessage;
    }

    public void setErrorMessage(String errorMessage) {
        this.errorMessage = errorMessage;
    }
}

File: data/src/main/java/br/albuquerque/data/dto/ResponseFavorite.kt
package br.albuquerque.data.dto

import com.google.gson.annotations.SerializedName

class ResponseFavorite(
    val error: String? = null,
    @SerializedName("error_message") val errorMessage: String? = null,
    val status: String? = null,
    val message: String? = null
)